pkgs/by-name/me/meson/package.nix

```diff
@@ -69,6 +69,10 @@ python3.pkgs.buildPythonApplication rec {
     # Fix cross-compilation of proc-macro (and mesa)
     # https://github.com/mesonbuild/meson/issues/12973
     ./0001-Revert-rust-recursively-pull-proc-macro-dependencies.patch
+
+    # Fix compilation of Meson using Ninja 1.12
```

Comment:
Why vendoring this patch?

Reply:
Because fetching the patch directly from https://github.com/mesonbuild/meson/commit/1b9eb6f3134d61c5ea0026ac95d807aab812c414 won't work as it depends on previous commits.

The issue Is exactly with patching file `unittests/baseplatformtests.py`. As you can see in the above discussion.

To ease the patching process, I manually cherry-picked and squashed the changes into a patch.